tests/conftest.py

```diff
@@ -28,30 +30,44 @@ def _test_env(monkeypatch):
 def base_workflow_instance(item_metadata, metadata_mapping, mocked_s3):
     class TestBaseWorkflow(BaseWorkflow):
 
-        def batch_metadata_iter(self):
+        def item_metadata_iter(self):
             yield from [item_metadata]
 
         def get_item_identifier(self, item_metadata):
             return item_metadata["item_identifier"]
 
         def get_bitstream_uris(self, item_identifier):
             bitstreams = [
-                "s3://dsc/workflow/folder/123_01.pdf",
-                "s3://dsc/workflow/folder/123_02.pdf",
-                "s3://dsc/workflow/folder/456_01.pdf",
+                "s3://dsc/base/batch-aaa/123_01.pdf",
+                "s3://dsc/base/batch-aaa/123_02.pdf",
+                "s3://dsc/base/batch-aaa/456_01.pdf",
```

Comment:
I think seeing the simulated filepaths in S3 here is nice!  Can imagine how this will work well over time.